tasks/definitions/container/attach.js

```diff
@@ -0,0 +1,21 @@
+const { spawnCmd } = require('@keg-hub/spawn-cmd')
+const { values } = require('../constants/values')
+
+/**
+ * Attach to watchtower container
+ * @param {Object} args 
+ */
+const attach = async () => spawnCmd(
+  `docker attach ${values.containerName}`
+)
+
+module.exports = {
+  attach: {
+    name: 'attach',
+    action: attach,
+    example: 'keg watchtower att',
+    description : 'Attaches to the watchtower container.',
+    alias: ['att', 'init'],
+    options: {}
```

Comment:
It may be helpful to add a workdir option, to set the starting directory when attaching to the container

Reply:
yeah if the `watchtower` container could actually be entered, I would agree. But because it's run as a command (it uses `ENTRYPOINT` directive), I don't think you can attach with a shell at all. I tried both `bash` and `sh` (like we did with the `traefik` container), nothing worked.

Reply:
Ahh, you would have to add the `--entrypoint` option as well, to override the default [docker docs](https://docs.docker.com/engine/reference/run/#entrypoint-default-command-to-execute-at-runtime). 
But in this case because it's not really something we need don't worry about it.